tools/make-man-page.sh

```diff
@@ -2,8 +2,15 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 set -ex
 
-pandoc -t man -s -o mkosi/resources/man/mkosi.1 mkosi/resources/man/mkosi.1.md
-pandoc -t man -s -o mkosi/resources/man/mkosi-addon.1 mkosi/resources/man/mkosi-addon.1.md
-pandoc -t man -s -o mkosi/resources/man/mkosi-initrd.1 mkosi/resources/man/mkosi-initrd.1.md
-pandoc -t man -s -o mkosi/resources/man/mkosi-sandbox.1 mkosi/resources/man/mkosi-sandbox.1.md
-pandoc -t man -s -o mkosi/resources/man/mkosi.news.7 mkosi/resources/man/mkosi.news.7.md
+EXTRA_ARGS=
+if [ -f mkosi/resources/pandoc/md2man.lua ]; then
+    EXTRA_ARGS="--lua-filter=mkosi/resources/pandoc/md2man.lua"
+else
+    echo "Warning: 'md2man.lua' was not found. Output may be degraded."
+fi
+
+pandoc ${EXTRA_ARGS} -t man -s -o mkosi/resources/man/mkosi.1 mkosi/resources/man/mkosi.1.md
```

Comment:
Style nit: No unnecessary curly braces
```suggestion
pandoc $EXTRA_ARGS -t man -s -o mkosi/resources/man/mkosi.1 mkosi/resources/man/mkosi.1.md
```